base_tier_validation/models/tier_validation.py

```diff
@@ -134,25 +142,15 @@ def _compute_reviewer_ids(self):
     def _search_validated(self, operator, value):
         assert operator in ("=", "!="), "Invalid domain operator"
```

Comment:
Really small change, but I would also add the deprecation note here (and `_search_rejected`). Migrations can be quite mechanical, but if you remove the field and leave the function it will be extra-confusing and risk to get a "delete at migration 20.0" comment.